packages/charts/backend/sirius-components-collaborative-charts/src/main/java/org/eclipse/sirius/components/collaborative/charts/CreateHierarchyEventHandler.java

```diff
@@ -88,6 +89,7 @@ public boolean canHandle(IEditingContext editingContext, IInput input) {
     }
 
     @Override
+    @Timed(value = Monitoring.EVENT_HANDLER)
```

Comment:
Why do we need to modify around 100 files to add those kind of timers if we monitor the time taken by the execution of an input in `EditingContextEventProcessor#handle`? I don't think we will spend that much time iterating on the representation event processors to find the right one and on the event handlers to find the right one too.